src/app/client/page.tsx

```diff
@@ -30,8 +31,8 @@ async function getAllTasks(token: string): Promise<TaskResponse[]> {
 }
 
 async function getAssigneeList(token: string): Promise<IAssignee> {
-  const res = await fetch(`${apiUrl}/api/users/client?token=${token}`, {
-    next: { tags: ['getAssigneeList'] },
+  const res = await fetch(`${apiUrl}/api/users/client?token=${token}&limit=${MAX_FETCH_ASSIGNEE_COUNT}`, {
+    next: { tags: ['getAssigneeList'], revalidate: ASSIGNEE_REVALIDATION_INTERVAL },
```

Comment:
I think using revalidation on the individual fetch doesn't make any difference since we are using `export const fetchCache = 'force-no-store'` and it affects all the fetch requests in this server component so revalidation parameter for individual fetch request doesn't work.